docs/data/sdks/typescript-react-native/migration.md

```diff
@@ -3,6 +3,254 @@ title: React Native SDK Migration Guide
 description: Use this guide to easily migrate from Amplitude's maintenance React Native SDK (@amplitude/react-native) to the new SDK (@amplitude/analytics-react-native).
 ---
 
+!!! warning
+    The Maintenance React Native SDK stores device ID, user ID, etc., in SQLite. However, there is no migration logic available (at this time), so this data won't be automatically migrated when upgrading to the new React Native SDK. Consequently, for example, device ID may be regenerated when using the new SDK.
+
+    We will release a migration plugin soon.
+
+Amplitude new React Native SDK (`@amplitude/analytics-react-native`) features a plugin architecture, built-in type definition and broader platform support. New React Native SDK isn't backwards compatible with maintenance React Native SDK `@amplitude/react-native`. 
```

Comment:
nit: I think we should stick to "latest" for consistency
```suggestion
Amplitude's latest React Native SDK (`@amplitude/analytics-react-native`) features a plugin architecture, built-in type definition and broader platform support. New React Native SDK isn't backwards compatible with maintenance React Native SDK `@amplitude/react-native`. 
```

Reply:
Also updated terminology [here](https://pr-819.d19s7xzcva2mw3.amplifyapp.com/data/sdks/typescript-react-native/migration/#terminology).